executor/write_test.go

```diff
@@ -261,7 +261,7 @@ func (s *testSuite) TestInsert(c *C) {
 	// issue 6424
 	tk.MustExec("drop table if exists t")
 	tk.MustExec("create table t(a time(6))")
-	tk.MustExec("insert into t value('20070219173709.055870'), ('20070219173709.055'), ('-20070219173709.055870'), ('20070219173709.055870123')")
+	tk.MustExec("insert into t value('20070219173709.055870'), ('20070219173709.055'), ('20070219173709.055870'), ('20070219173709.055870123')")
```

Comment:
Insert some negative cases is proper (the first and the third case are same), it's better to change the L265.

Reply:
If there is an invalid string for `Duration`, this `SQL` statement will fail so does the `tk.MustExec` function.